src/views/Visualization.vue

```diff
@@ -78,6 +68,32 @@ export default {
       Elevation: () => import( /* webpackPrefetch: true */ /*webpackChunkName: "section"*/ "./../components/Elevation"),
       References: () => import( /* webpackPrefetch: true */ /*webpackChunkName: "References"*/ "./../components/References"),
       Chapter
+    },
+    mounted(){
+      //Waits until the first DOM paint is complete
+      //Allows us to only run the functionality when we know the DOM elements are loaded
+      this.$nextTick(() => {
+        this.parallaxScroll();
+      })
+    },
+    methods:{
+      parallaxScroll(){
+        const gsap = this.$gsap;
+        gsap.registerPlugin(ScrollToPlugin, ScrollTrigger); // register gsap plugins for scrollTrigger
+        gsap.utils.toArray(".parallax").forEach((chapter, i) => {
+          chapter.bg = chapter.querySelector(".bg");
+          gsap.to(chapter.bg, {
+            ease: "none",
+            scrollTrigger: {
+              trigger: chapter,
+              start: "bottom bottom",
+              scrub: true,
```

Comment:
👀 I'm into the scrub effect

Reply:
It does really make cool effects.  Especially when you can figure out a way to use the timing aspect of it.